addons/l10n_es_modelo130/data/mod130.xml

```diff
@@ -0,0 +1,323 @@
+                                <field name="date_scope">from_fiscalyear</field>
+                            </record>
+                            <record id="mod_130_casilla_06_balance" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">domain</field>
+                                <field name="formula" eval="[
+                                    ('tax_tag_ids', '=', '-mod130[06]'),
+                                    '|',
+                                    ('partner_id.industry_id', '=', False),
+                                    ('partner_id.industry_id.name', '!=', 'Agriculture')
+                                 ]"/>
+                                <field name="subformula">sum</field>
+                                <field name="date_scope">from_fiscalyear</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="mod_130_casilla_07" model="account.report.line">
+                        <field name="name">[07] Box 04 - Box 05 - Box 06</field>
+                        <field name="code">aeat_mod_130_07</field>
+                        <field name="expression_ids">
+                            <record id="mod_130_casilla_07_balance" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">aeat_mod_130_04.balance - aeat_mod_130_05.balance - aeat_mod_130_06.balance</field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id="mod_130_title_2" model="account.report.line">
+                <field name="name">II. Agriculture, livestock farming, forestry and fishing in the direct assessment system, normal or simplified mode.</field>
+                <field name="code">aeat_mod_130_title_2</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="mod_130_casilla_08" model="account.report.line">
+                        <field name="name">[08] Revenue</field>
+                        <field name="code">aeat_mod_130_08</field>
+                        <field name="groupby">account_id</field>
+                        <field name="foldable" eval="True"/>
+                        <field name="expression_ids">
+                            <record id="mod_130_casilla_08_balance" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">domain</field>
+                                <field name="formula" eval="
```

Comment:
we need to be consitant here, sometimes it's all in one line, sometime you put the [ after the eval and go to the next line for the domain. I think we can choose one way and stick with it :smile: